rllib/algorithms/bc/tests/test_bc_new_stack.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+import unittest
+import ray
+
+from ray.rllib.algorithms.bc import BCConfig
+from ray.rllib.utils.metrics import ENV_RUNNER_RESULTS, EPISODE_RETURN_MEAN
+
+
+class TestBC(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        ray.init()
+
+    @classmethod
+    def tearDownClass(cls) -> None:
+        ray.shutdown()
+
+    def test_bc_compilation_and_learning_from_offline_file(self):
+        data_path = "tests/data/cartpole/large.json"
+        base_path = Path(__file__).parents[3]
+        print(f"base_path={base_path}")
+        data_path = "local://" + base_path.joinpath(data_path).as_posix()
+        print(f"data_path={data_path}")
+
+        # Define the BC config.
+        config = (
+            BCConfig()
```

Comment:
So cool to see this in action!!!

Reply:
Yeah, it took a time to make it run, but now it runs pretty smoothly.

Reply:
That's awesome!!